victims/models.py

```diff
@@ -27,6 +29,20 @@ def is_valid(self):
 
         return True
 
+    def to_JSON(self):
```

Comment:
`to_JSON` is a syntax that [does not follow PEP8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables):

> **Method Names and Instance Variables**
> 
> Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.

Can we rename it to `to_json`?